pkg/sql/set_cluster_setting.go

```diff
@@ -61,6 +61,8 @@ type setClusterSettingNode struct {
 	value tree.TypedExpr
 }
 
+var SettingOverrideErr = errors.New("cluster setting is overridden by system virtual cluster")
```

Comment:
super nit: could this error live in `pkg/settings`, so we minimize needing to import the large `sql` package?

Reply:
Done.